lnst/External/TRex/UDPSimple.py

```diff
@@ -23,8 +23,8 @@ def _import_optionals():
 
     def create_stream (self, **kwargs):
         # Use port's configured mac and ip addresses
-        L2 = Ether()
-        L3 = IP()
+        L2 = Ether(src=kwargs["src_mac"], dst=kwargs["dst_mac"])
+        L3 = IP(src=kwargs["src_ip"], dst=kwargs["dst_ip"])
```

Comment:
Are these always present in kwargs?

Reply:
yes, this is where these get generated: https://github.com/LNST-project/lnst/blob/master/lnst/External/TRex/TRexLib.py#L96C28-L96C44

and these in turn come from the mandatory flows parameter here: https://github.com/LNST-project/lnst/blob/master/lnst/Tests/TRex.py#L13

Reply:
Ok, thanks for clarifying this.